src/test/java/org/corfudb/infrastructure/LayoutServerTest.java

```diff
@@ -148,8 +155,8 @@ public void checkLayoutPersisted()
         LayoutServer s1 = getDefaultServer(serviceDir);
         bootstrapServer(TestLayoutBuilder.single(9000));
         Layout l100 = TestLayoutBuilder.single(9000);
-        l100.setEpoch(100);
-        sendMessage(new LayoutRankMsg(null, 100, CorfuMsgType.LAYOUT_PREPARE));
+        //l100.setEpoch(100);
```

Comment:
I will add more comments in general to all the tests. This one here is checking if phase1 and phase2 get persisted such that we still have data after reboot. I will add more comments to all the tests.